### PR TITLE
Create accountInformation.schema.tpl.json for Person

### PR DESCRIPTION
**Pull request by lzehl**
related to #368

### CODE REVIEW

File: schemas/actors/accountInformation.schema.tpl.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "_type": "https://openminds.ebrains.eu/core/AccountInformation",
+  "required": [
+    "service",
+    "userName"
+  ],
+   "properties": {
+     "service": {
+       "_instruction": "Add the web service (version) of this account.",
```
Comment (apdavison) [inline schemas/actors/accountInformation.schema.tpl.json]: the instruction mentions the web service version, but the linked type is WebService. I think the latter is more appropriate, so the instruction should be changed.

File: schemas/actors/person.schema.tpl.json [hunk 2]
```diff
@@ -7,6 +7,15 @@
     "givenName"
   ],
   "properties": {
+    "accountInformation": {
```
Comment (apdavison) [inline schemas/actors/person.schema.tpl.json]: I think "associatedAccount" might be a better term

File: schemas/actors/person.schema.tpl.json [hunk 3]
```diff
@@ -7,6 +7,15 @@
...
+      "type": "array",
+      "minItems": 1,
+      "uniqueItems": true,
+      "_instruction": "Add one or several account information for this person.",
```
Comment (apdavison) [inline schemas/actors/person.schema.tpl.json]: Maybe "Add information about web service accounts held by this person."

File: schemas/actors/accountInformation.schema.tpl.json (hunk 1 above)
Comment (lzehl) [reply schemas/actors/accountInformation.schema.tpl.json]: I agree. sorry this was a leftover when I was thinking to link both...

**Comment by lzehl** [comment]
@apdavison should the schema name remain AccountInformation ? or should that be change to something else ?

**Comment by apdavison** [comment]
@lzehl yes, I think AccountInformation is accurate.

**Review by apdavison — APPROVED** [review]

**Comment by lzehl** [comment]
@olinux this merge requires an update in the EBRAINS KG we should discuss.

**Pull request merged**